nextjs_frontend/src/app/proposal/[cpfDevedor]/components/ProposalCard/proposal-card.tsx

```diff
@@ -0,0 +1,68 @@
+import { Acordo } from "@/models/Acordos";
+import Styles from "./proposal-card.module.scss";
+
+interface ProposalCardProps {
+  proposal: Acordo;
+  position: number;
+  isSelected: boolean;
+  onSelected: (proposalID: number) => void;
+}
+
+export default function ProposalCard({
+  proposal,
+  position,
+  isSelected,
+  onSelected,
+}: ProposalCardProps) {
+  function formatJuros(juros: number) {
+    return juros !== 0 ? `${juros * 100}%` : "Sem taxa de juros";
+  };
+
+  function totalValue() {
+    return (proposal.valor / proposal.qtdParcelas)
+            .toFixed(2).replace(".", ",")
+  }
```

Comment:
Shouldn't it be called installmentsValue?

Reply:
up